app/importers/planning_applications_importer.rb

```diff
@@ -48,21 +48,28 @@ def import_rows(filename:)
   end
 
   def import_row(row)
-    with_property(row) do |property|
-      PlanningApplication.find_or_initialize_by(reference: row[:reference])
-                         .update!(
-                           reference: row[:reference],
-                           area: row[:area],
-                           proposal_details: row[:proposal_details],
-                           received_at: row[:received_at],
-                           officer_name: row[:officer_name],
-                           decision: row[:decision],
-                           decision_issued_at: row[:decision_issued_at],
-                           view_documents: row[:view_documents],
-                           property: property,
-                           local_authority: local_authority
-                         )
-    end
+    PlanningApplicationCreation.new(
+      local_authority: local_authority,
+      reference: row[:reference],
+      area: row[:area],
+      proposal_details: row[:proposal_details],
+      received_at: row[:received_at],
+      officer_name: row[:officer_name],
+      decision: row[:decision],
+      decision_issued_at: row[:decision_issued_at],
+      view_documents: row[:view_documents],
+      uprn: row[:uprn],
+      property_code: row[:property_code],
+      property_type: row[:property_type],
+      full: row[:full],
+      address: row[:address],
+      town: row[:town],
+      postcode: row[:postcode],
+      map_east: row[:map_east],
+      map_north: row[:map_north],
+      ward_code: row[:ward_code],
+      ward_name: row[:ward_name],
+    ).perform
```

Comment:
Would `PlanningApplicationCreation.new(**row.merge(local_authority: local_authority))` work here?

Reply:
Yeah I could, just thought is better to explicitly pass each attribute. Happy to change it

Reply:
I've tried this change and some specs are failing so I will leave it as it is